### DISCUSSION TITLE
ASRock B650 Steel Legend: 0-6% and 0-26% dead zones on case fans

### DISCUSSION THREAD

**Discussion by twest820** [Rem0o/FanControl.Releases#2743] [Q&A] [answered]
Just brought up a Win11 23H2 build on a B650 Steel Legend with the 3.06 BIOS (latest one, AGESA 1.2.0.0a as ASRock doesn't have a 1.2.0.2 version out yet) and dropped v203 on it. Either it's got some quirks or the 9900X (#2733) is somehow introducing oddities.

* CHA_FAN1: aliases 0-6% from FanControl to 0%, 7% constantly jumps between 7 and 15%, fully responsive at 8+%
* CHA_FAN2, 3, 4: alias 0-26% from FanControl to 0%, fully responsive at 27+%
* CPU_FAN1 and CPU_FAN2/WP: fully responsive at all levels

Anyone got ideas for how to bust CHA_FAN2-4 loose so control signals in ~5-26% range are honored? The fans are fully responsive to percentages in this when under BIOS control, so it seems like there's an interop issue between the board and FanControl. SMBus drivers are AMD's current ones.

BIOS settings seem to have no effect on the size of the dead zone whatsoever. I've tried multiple types of BIOS fan curves, including the recommended flat 50%. Auto, DC, and PWM control. Fan stop enabled and disabled, several different gate settings, and multiple threshold settings.

CHA_FAN1 is not a problem as I don't need below 10% from it but 27% minimum's not great for taking channels 2-4 in and out of fan stop or reducing dust at light loads. 27% on the Arctic P12s I'm using yields a ~630 RPM minimum in both DC and PWM. (The speed reduction resistors I have handy actually end up increasing this to 730 and 920 RPM, respectively. Yay.)

**Comment by Rem0o** [comment, marked as answer]
Isn't that just standard start/stop% ?   It is normal a fan has a deadzone in the lowest range and minimum % for it to kick start.  You can't run most 120mm+ fans below like 400-500 rpm. 

Most fans have a higher start% like 30 just to get it going, but then you may go down below that value while it is still spinning. 

FanControl auto detects those deadzones and sets very safe/conservative values that will negate some of the fans lowest range to be safe.

Just expand your control cards and see what is set for start and stop% for every fan.

**Reply by twest820** [reply]
You're right, this is another manifestation of the [stop-start regression](https://github.com/Rem0o/FanControl.Releases/discussions/2333#discussioncomment-9002264) introduced by RPM calibration. Thanks, I'd forgotten about that.

The correct start value for a P12's ~5% and the stop's ~7% but FanControl decided to use 27% for start and put the stops to 35 or 36%. I understand the motivation here (I do much the same thing in setting fan curves) but would say value chosen are conservative by 20–25%. So far across all the fans and mobos I've worked with I've not needed more than a 1–2% margin.

My experience is fans with 30+% start points are uncommon. Most of the models I usually work with start in the 5–20% range, give or take a percent or two for fan to fan and mobo to mobo variation.
